tests/lib/util.py

```diff
@@ -48,9 +48,27 @@ def create_commit(
     **kwargs,
 ) -> str:
     options = ""
+
     if dt is not None:
+        # Store committer date in case it was set somewhere else
+        original_committer_date = os.environ.get("GIT_COMMITTER_DATE", None)
+
         options += f"--date {dt.isoformat()}"
-    return execute(cwd, f'git commit -m "{message}" {options}', **kwargs)
+        # The committer date is what is used to determine sort order for tags, etc
+        os.environ["GIT_COMMITTER_DATE"] = f"--date {dt.isoformat()}"
```

Comment:
`GIT_COMMITER_DATE` should contain date, not `--date $value`
https://git-scm.com/docs/git-commit/2.24.0#_date_formats

Reply:
Yes, yes it should.  Now I'm doubting the testing I did.  So confused...